tests/openwisp2/celery.py

```diff
@@ -0,0 +1,11 @@
+import os
```

Comment:
This file can moved to `openwisp_radius.celery` and imported in `__init__.py` from there, I don't know why but the same is not done in the openwisp-controller and openwisp-monitoring.

The `__init__.py` can have `os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'openwisp2.settings')`.

Is the reason for `celery.py` to be outside the app is that we don't want `DJANGO_SETTINGS_MODULE` in `__init__.py` or is there another reason for it? :smile:

Reply:
I think we're just following the guidelines of celery when integrating in django, if you find different guidelines let me know